.github/workflows/contribution_evaluation.yaml

```
@@ -0,0 +1,215 @@
+      - name: Run workload analysis script
+        id: analyze_workload
+        run: |
+          python scripts/workload_analyzation.py \
+            --owner ${{ github.repository_owner }} \
+            --repo ${{ github.event.repository.name }} \
+            --pr-number ${{ github.event.number }} \
+            --token ${{ secrets.GITHUB_TOKEN }}
+            
+          workload=$(grep "Estimated workload" workload_output.txt | awk -F': ' '{print $2}')
+          changedLines=$(grep "Estimated changedLine" workload_output.txt | awk -F': ' '{print $2}')
+          
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)